### ISSUE TITLE
issue with cue files in folders

### ISSUE THREAD

**Issue by cueckoo** [cue-lang/cue#286]
_Originally opened by @cameronbraid in https://github.com/cuelang/cue/issues/286_

### What version of CUE are you using (`cue version`)?

built from git master 11 Feb 2020

### Does this issue reproduce with the latest release?
yes

### What operating system are you using?
linux

### What did you do?
https://github.com/cameronbraid/cue-nested-broken/

```
.
├── nested-broken
│   ├── a
│   │   └── a.cue
│   ├── b.cue
│   ├── cue.mod
│   │   ├── module.cue
│   │   ├── pkg
│   │   └── usr
│   ├── dump_tool.cue
│   ├── foo_tool.cue
│   ├── ls_tool.cue
│   └── schema.cue
├── nested-working
│   ├── a
│   │   └── a.cue
│   ├── b
│   │   └── b.cue
│   ├── cue.mod
│   │   ├── module.cue
│   │   ├── pkg
│   │   └── usr
│   ├── dump_tool.cue
│   ├── foo_tool.cue
│   ├── ls_tool.cue
│   └── schema.cue
└── run.sh
```

```
# ./run.sh
nested-working
name: a
---
name: b

nested-broken
text: invalid operation !x.disabled (! _|_):
    ./foo_tool.cue:4:25
```

The only difference in nested-broken is that b.cue is in the parent folder.

**Comment by cueckoo** [comment]
_Original reply by @mpvl in https://github.com/cuelang/cue/issues/286#issuecomment-590275359_

If someone could investigate this with a more precise error analysis that would be great.

**Comment by cueckoo** [comment]
_Original reply by @mpvl in https://github.com/cuelang/cue/issues/286#issuecomment-702175663_

After running `cue fix` with v0.2.2, this works on tip.
